planner_cspace/include/planner_cspace/grid_astar.h

```diff
@@ -313,6 +313,8 @@ class GridAstar
         }
         if (centers.size() < 1 || found)
           break;
+        updates.clear();
+        dont.clear();
```

Comment:
Variance of cost estimation cache update is increased if this is added. Path plan duration is decreased of-course.